e2e/tests/testdata/b87e7c2782837a3538b0aa3dceb9e8133765ec81_stable-4.3_amd64-production.json

```diff
@@ -0,0 +1,259 @@
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2,candidate-4.3,fast-4.3,stable-4.3",
+        "io.openshift.upgrades.graph.release.manifestref": "sha256:dfbe59ca5dcc017475a0e1c703f51750c1bde63f12c725fbe4b7a599e36eb725",
+        "url": "https://access.redhat.com/errata/RHBA-2020:0826"
+      }
+    },
+    {
+      "version": "4.2.26",
+      "payload": "quay.io/openshift-release-dev/ocp-release@sha256:af0a6384e787c820279a0954c804d9c27b13458df2de366b3cd5ec3a7cdaa4b2",
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2,candidate-4.3,fast-4.3,stable-4.3",
+        "io.openshift.upgrades.graph.release.manifestref": "sha256:af0a6384e787c820279a0954c804d9c27b13458df2de366b3cd5ec3a7cdaa4b2",
+        "url": "https://access.redhat.com/errata/RHBA-2020:0936"
+      }
+    },
+    {
+      "version": "4.2.27",
+      "payload": "quay.io/openshift-release-dev/ocp-release@sha256:26b501f5111add85989de9f02e368a0bfda8b43a54d0d56f4bf5916abada34da",
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2,candidate-4.3,fast-4.3,stable-4.3",
+        "io.openshift.upgrades.graph.release.manifestref": "sha256:26b501f5111add85989de9f02e368a0bfda8b43a54d0d56f4bf5916abada34da",
+        "url": "https://access.redhat.com/errata/RHBA-2020:1263"
+      }
+    },
+    {
+      "version": "4.2.28",
+      "payload": "quay.io/openshift-release-dev/ocp-release@sha256:c196196d04e08518f0b1ea2926c07643b4a0fd0c0b8940c1d541dacd803a6029",
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2,candidate-4.3,fast-4.3,stable-4.3",
+        "io.openshift.upgrades.graph.release.manifestref": "sha256:c196196d04e08518f0b1ea2926c07643b4a0fd0c0b8940c1d541dacd803a6029",
+        "url": "https://access.redhat.com/errata/RHBA-2020:1398"
+      }
+    },
+    {
+      "version": "4.2.29",
+      "payload": "quay.io/openshift-release-dev/ocp-release@sha256:3bff53ce2202ec59ed87581106b05f364fea0e7459f5806e4dc6e5129f130b36",
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.release.channels": "candidate-4.2,fast-4.2,stable-4.2,candidate-4.3,fast-4.3,stable-4.3",
+        "io.openshift.upgrades.graph.release.manifestref": "sha256:3bff53ce2202ec59ed87581106b05f364fea0e7459f5806e4dc6e5129f130b36",
+        "url": "https://access.redhat.com/errata/RHBA-2020:1450"
+      }
+    },
+    {
+      "version": "4.3.0",
+      "payload": "quay.io/openshift-release-dev/ocp-release@sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d",
+      "metadata": {
+        "description": "",
+        "io.openshift.upgrades.graph.previous.remove_regex": "4.2.16",
```

Comment:
This doesn't match https://github.com/openshift/cincinnati-graph-data/blob/b87e7c2782837a3538b0aa3dceb9e8133765ec81/blocked-edges/4.3.0.yaml#L2